BankexWallet/InterfaceElements/DesignElements/Onboarding/CustomNetworkViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  CustomNetworkViewController.swift
+//  BankexWallet
+//
+//  Created by Георгий Фесенко on 01/06/2018.
+//  Copyright © 2018 Alexander Vlasov. All rights reserved.
+//
+
+import UIKit
+
+class CustomNetworkViewController: UIViewController {
```

Comment:
Very bad naming. It's controller from onboarding with some tips/ info, but I cannot understand it from the name. BTW, do you really need several separate controllers for onboarding? Can't you just have an array of images/texts and only one real controller to show this info?

Reply:
Yes, I renamed the controller and also removed all other view controllers.